### ISSUE TITLE
Chromosome not labeled correctly? 

### ISSUE THREAD

**Issue by ahdee** [stianlagstad/chimeraviz#83]
Hi, 
I'm using version chimeraviz_1.13.8. 
There is strange error I'm getting whereby the labeling of the chromosomes are not labeled correctly? 

For example here is a fusion 
```
[1] "Fusion object"
[1] "id: 4"
[1] "Fusion tool: starfusion"
[1] "Genome version: hg38"
[1] "Gene names: NSD1-RHOG"
[1] "Chromosomes: chr5-chr11"
[1] "Strands: +,-"
[1] "In-frame?: FALSE"
```

so here it should be labled as chr5 and 11 however ths is what I get instead when I try to plot it. 

```
  plot_fusion(
    fusion = draw.this,
    edb = edb,
    reduce_transcripts = T
    )
  
```

attached is an rds for the draw.this object I used to plot. 
https://www.dropbox.com/s/ihuyz57oz9sx2oq/test.rds?dl=1

![cm6Hnqm 1](https://user-images.githubusercontent.com/9808122/80287153-b07f0000-86e4-11ea-91e1-dd1937237a37.png)

```
R version 3.6.2 (2019-12-12)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 10 (buster)

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/libopenblasp-r0.3.5.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C              LC_PAPER=en_US.UTF-8      
 [8] LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
 [1] grid      splines   stats4    parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] chimeraviz_1.13.8           GSVA_1.34.0                 corrplot_0.84               parallelDist_0.2.4          mrfDepth_1.0.11             DrInsight_0.1.1            
 [7] qusage_2.20.0               igraph_1.2.5                ensembldb_2.10.2            AnnotationFilter_1.10.0     GenomicFeatures_1.38.2      Gviz_1.30.3                
[13] Biostrings_2.54.0           XVector_0.26.0              univOutl_0.1-5              Hmisc_4.4-0                 Formula_1.2-3               robustbase_0.93-6          
[19] moments_0.14                org.Hs.eg.db_3.10.0         AnnotationDbi_1.48.0        heatmap3_1.1.7              ggrepel_0.8.2               PerformanceAnalytics_2.0.4 
[25] xts_0.12-0                  zoo_1.8-7                   d3heatmap_0.6.1.2           dendextend_1.13.4           wesanderson_0.3.6           vegan_2.5-6                
[31] lattice_0.20-38             permute_0.9-5               FactoMineR_2.3              factoextra_1.0.6            edgeR_3.28.1                gtable_0.3.0               
[37] forestplot_1.9              checkmate_2.0.0             data.table_1.12.8           ggbeeswarm_0.6.0            survminer_0.4.6.999         survival_3.1-11            
[43] karyoploteR_1.12.4          regioneR_1.18.1             gtools_3.8.1                garnett_0.2.11              shiny_1.4.0.2               monocle3_0.2.1             
[49] SingleCellExperiment_1.8.0  SummarizedExperiment_1.16.1 DelayedArray_0.12.2         BiocParallel_1.20.1         matrixStats_0.56.0          GenomicRanges_1.38.0       
[55] GenomeInfoDb_1.22.0         IRanges_2.20.2              S4Vectors_0.24.3            leidenbase_0.1.0            gridExtra_2.3               fgsea_1.12.0               
[61] Rcpp_1.0.4                  ComplexHeatmap_2.2.0        reshape2_1.4.3              feather_0.3.5               patchwork_1.0.0             monocle_2.14.0             
[67] DDRTree_0.1.5               irlba_2.3.3                 VGAM_1.1-2                  Biobase_2.46.0              BiocGenerics_0.32.0         ggridges_0.5.2             
[73] openxlsx_4.1.4              limma_3.42.2                ggpubr_0.2.5                magrittr_1.5                forcats_0.5.0               stringr_1.4.0              
[79] purrr_0.3.3                 readr_1.3.1                 tidyr_1.0.2                 tibble_2.1.3                ggplot2_3.3.0               tidyverse_1.3.0            
[85] kableExtra_1.1.0            knitr_1.28                  biomaRt_2.42.1              cowplot_1.0.0               RColorBrewer_1.1-2          Matrix_1.2-18              
[91] dplyr_0.8.5                 Seurat_3.1.4                BiocManager_1.30.10        

loaded via a namespace (and not attached):
  [1] pbapply_1.4-2            haven_2.2.0              vctrs_0.2.4              usethis_1.5.1            fastICA_1.2-2            mgcv_1.8-31              blob_1.2.1              
  [8] later_1.0.0              DBI_1.1.0                rappdirs_0.3.1           uwot_0.1.8               jpeg_0.1-8.1             zlibbioc_1.32.0          htmlwidgets_1.5.1       
 [15] mvtnorm_1.1-0            GlobalOptions_0.1.1      future_1.16.0            leaps_3.1                leiden_0.3.3             DEoptimR_1.0-8           KernSmooth_2.23-16      
 [22] DT_0.13                  promises_1.1.0           gdata_2.18.0             pkgload_1.0.2            graph_1.64.0             RcppParallel_5.0.0       fs_1.3.2                
 [29] fastmatch_1.1-0          mnormt_1.5-6             digest_0.6.25            png_0.1-7                qlcMatrix_0.9.7          sctransform_0.2.1        pkgconfig_2.0.3         
 [36] docopt_0.6.1             estimability_1.3         reticulate_1.14          circlize_0.4.8           beeswarm_0.2.3           GetoptLong_0.1.8         xfun_0.12               
 [43] tidyselect_1.0.0         ica_1.0-2                viridisLite_0.3.0        rtracklayer_1.46.0       pkgbuild_1.0.6           rlang_0.4.5              glue_1.3.2              
 [50] metap_1.3                modelr_0.1.6             emmeans_1.4.5            ggsignif_0.6.0           labeling_0.3             gbRd_0.4-11              mutoss_0.1-12           
 [57] httpuv_1.5.2             Rttf2pt1_1.3.8           TH.data_1.0-10           annotate_1.64.0          webshot_0.5.2            jsonlite_1.6.1           bit_1.1-15.2            
 [64] mime_0.9                 gplots_3.0.3             Rsamtools_2.2.3          BiocStyle_2.14.4         stringi_1.4.6            processx_3.4.2           quadprog_1.5-8          
 [71] bitops_1.0-6             cli_2.0.2                Rdpack_0.11-1            RSQLite_2.2.0            pheatmap_1.0.12          rstudioapi_0.11          org.Mm.eg.db_3.10.0     
 [78] GenomicAlignments_1.22.1 nlme_3.1-142             fastcluster_1.1.25       locfit_1.5-9.4           VariantAnnotation_1.32.0 listenv_0.8.0            survMisc_0.5.5          
 [85] dbplyr_1.4.2             sessioninfo_1.1.1        readxl_1.3.1             lifecycle_0.2.0          munsell_0.5.0            cellranger_1.1.0         caTools_1.18.0          
 [92] codetools_0.2-16         coda_0.19-3              magic_1.5-9              vipor_0.4.5              lmtest_0.9-37            htmlTable_1.13.3         lsei_1.2-0              
 [99] xtable_1.8-4             ROCR_1.0-7               flashClust_1.01-2        scatterplot3d_0.3-41     abind_1.4-5              farver_2.0.3             FNN_1.1.3               
[106] km.ci_0.5-2              RANN_2.6.1               askpass_1.1              biovizBase_1.34.1        sparsesvd_0.2            bibtex_0.4.2.2           RcppAnnoy_0.0.16        
[113] shinythemes_1.1.2        dichromat_2.0-0          cluster_2.1.0            future.apply_1.4.0       extrafontdb_1.0          ellipsis_0.3.0           prettyunits_1.1.1       
[120] lubridate_1.7.4          reprex_0.3.0             multtest_2.42.0          remotes_2.1.1            slam_0.1-47              TFisher_0.2.0            testthat_2.3.2          
[127] geometry_0.4.5           htmltools_0.4.0          BiocFileCache_1.10.2     yaml_2.2.1               plotly_4.9.2             XML_3.99-0.3             foreign_0.8-72          
[134] withr_2.1.2              fitdistrplus_1.0-14      bit64_0.9-7              multcomp_1.4-12          ProtGenerics_1.18.0      combinat_0.0-8           rsvd_1.0.3              
[141] devtools_2.2.2           waffle_0.7.0             bamsignals_1.18.0        memoise_1.1.0            evaluate_0.14            callr_3.4.3              geneplotter_1.64.0      
[148] extrafont_0.17           ps_1.3.2                 curl_4.3                 fansi_0.4.1              highr_0.8                GSEABase_1.48.0          acepack_1.4.1           
[155] desc_1.2.0               npsurv_0.4-0             rjson_0.2.20             rprojroot_1.3-2          clue_0.3-57              tools_3.6.2              sandwich_2.5-1          
[162] RCurl_1.98-1.1           ape_5.3                  bezier_1.1.2             xml2_1.2.5               httr_1.4.1               assertthat_0.2.1         rmarkdown_2.1           
[169] globals_0.12.5           R6_2.4.1                 nnet_7.3-12              progress_1.2.2           shape_1.4.4              colorspace_1.4-1         RCircos_1.2.1           
[176] generics_0.0.2           base64enc_0.1-3          pillar_1.4.3             sn_1.6-0                 HSMMSingleCell_1.6.0     GenomeInfoDbData_1.2.2   plyr_1.8.6              
[183] rvest_0.3.5              zip_2.0.4                latticeExtra_0.6-29      fastmap_1.0.1            broom_0.5.5              openssl_1.4.1            BSgenome_1.54.0         
[190] scales_1.1.0             backports_1.1.5          plotrix_3.7-7            densityClust_0.3         ArgumentCheck_0.10.2     hms_0.5.3                Rtsne_0.15              
[197] KMsurv_0.1-5             numDeriv_2016.8-1.1      lazyeval_0.2.2           tsne_0.1-3               crayon_1.3.4             MASS_7.3-51.4            viridis_0.5.1           
[204] rpart_4.1-15             compiler_3.6.2   
```

**Comment by stianlagstad** [comment]
Hi @ahdee! Thank you for the report. Are you able to share the starfusion results file that you have imported? Then I can see if I can reproduce the error.

**Comment by ahdee** [comment]
@stianlagstad  sure thing, thanks! This is a file run with starfusion, however I tried it with soap, eric  and both give the same strange incorrect annotations! 
https://www.dropbox.com/s/vkc8l5gai4uqayh/rhog.tsv?dl=1

**Comment by stianlagstad** [comment]
Thank you very much @ahdee ! I located the issue and am fixing it here: https://github.com/stianlagstad/chimeraviz/pull/84. The fix will be included in version 1.13.9 of chimeraviz.

**Comment by ahdee** [comment]
awesome thank you @stianlagstad !
